src/eos-updater-poll-common.c

```diff
@@ -322,6 +322,7 @@ get_refspec_to_upgrade_on_from_deployment (OstreeSysroot     *sysroot,
   g_autoptr(GVariant) ref_for_deployment_variant = NULL;
   const gchar *refspec_for_deployment = NULL;
   g_autoptr(OstreeRepo) repo = NULL;
+  g_autoptr(GError) local_error = NULL;
```

Comment:
This commit (`poll: Ignore the checkpoint refspec if it didn't parse correctly`) looks good to me, and I can’t see any issues with it. Please squash it into the commit which introduces checkpoint support. Thanks.

Reply:
Done